lib/node_modules/@stdlib/math/base/complex/exp/README.md

```diff
@@ -0,0 +1,117 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2018 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# exp
+
+> Compute the [exponential][exponential-function] function of a complex number.
+
+<section class="intro"> 
+
+The [exponential][exponential-function] function of a complex number is defined as
+
+<!-- <equation class="equation" label="eq:cexp_function" align="center" raw="\operatorname{exp}(z) = e^{x + i y} = (\exp{x}) (\cos(y) + i \sin(y))" alt="complex exponential function"> -->
+
+<!-- </equation> -->
+
+</section>
+
+<!-- /.intro -->
+
+<section class="usage">
+
+## Usage
+
+``` javascript
+var cexp = require( '@stdlib/math/base/complex/exp' );
+```
+
+#### cexp( \[out,\] re, im )
```

Comment:
FYI: the second escape is not necessary; only the first bracket must be escaped. Does not need to be changed, however, as this will be automatically updated when an equation is inserted.